Makefile

```diff
@@ -0,0 +1,104 @@
+# Makefile of the Arch Linux Updates Indicator GNOME Shell Extension
+# (adapted from https://gitlab.com/skrewball/openweather/)
+
+PKG_NAME = arch-update
+UUID = arch-update@RaphaelRochet
+BASE_MODULES = metadata.json LICENCE.txt prefs.xml
+SRC_MODULES = extension.js prefs.js stylesheet.css
+EXTRA_DIRECTORIES = icons
+TOLOCALIZE = extension.js prefs.js
+MSGSRC = $(wildcard locale/*/arch-update.po)
+
+# Packagers: Use DESTDIR for system wide installation
+ifeq ($(strip $(DESTDIR)),)
+	INSTALLTYPE = local
+	INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+else
+	INSTALLTYPE = system
+	SHARE_PREFIX = $(DESTDIR)/usr/share
+	INSTALLBASE = $(SHARE_PREFIX)/gnome-shell/extensions
+endif
+# Set a git version for self builds from the latest git tag with the revision
+# (a monotonically increasing number that uniquely identifies the source tree)
+# and the current short commit SHA1. (Note: not set if VERSION passed)
+GIT_VER = $(shell git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g')
+# The command line passed variable VERSION is used to set the version integer
+# in the metadata and in the generated zip file. If no VERSION is passed, we
+# won't touch the metadata version and instead use that for the zip file.
+ifdef VERSION
```

Comment:
Is is supposed to be set on command line ?

Reply:
Yes a shell variable, try `VERSION=99 make zip-file` or `VERSION=99 make install` and restart. The manifest isn't touched.